frontends/bsd/src/screens/write_in_adjudication_screen.tsx

```diff
@@ -498,27 +531,27 @@ export function WriteInAdjudicationScreen({
         prev
       )
     );
-    setSelectedContestId(undefined);
     await onAdjudicationComplete?.(sheetId, side, adjudications);
   }, [adjudications, onAdjudicationComplete, setWriteInPresets, sheetId, side]);
 
-  const selectedContestIdOrDefault = selectedContestId ?? contestIds[0];
-  const selectedContestIndex = contestIds.indexOf(selectedContestIdOrDefault);
-  const isFirstContestSelected = selectedContestIndex === 0;
+  const isFirstContestSelected =
+    selectedContestId === contestsWithWriteInsIds[0];
   const isLastContestSelected =
-    selectedContestIndex === contestsWithWriteInsCount - 1;
+    selectedContestId ===
+    contestsWithWriteInsIds[contestsWithWriteInsIds.length - 1];
 
-  const contestsWithWriteInsIndex = [...contestsWithWriteIns].findIndex(
-    (id) => id === selectedContestIdOrDefault
+  const selectedContestIdOrDefaultIndex = contestsWithWriteInsIds.indexOf(
+    selectedContestId
```

Comment:
I think this variable is equivalent to `selectedContestIndex` and can be removed